ang/test/civicase/case/list/directives/case-list-sort-header.directive.spec.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jasmine */
+
+describe('civicaseCaseListSortHeader directive', function () {
+  var element, $compile, $rootScope, scope;
+
+  beforeEach(module('civicase'));
+
+  beforeEach(inject(function (_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+    scope = $rootScope.$new();
+  }));
+
+  describe('basic tests', function () {
+    var header;
+
+    describe('when sortable is false', function () {
+      beforeEach(function () {
+        scope.sort = { field: '', dir: '', sortable: false };
+        header = {
+          display_type: 'activity_card',
+          label: 'Next Activity',
+          name: 'next_activity',
+          sort: 'next_activity'
+        };
+        compileDirective(scope, header);
```

Comment:
Add new line after var declarations. Check this at other places too

Reply:
I would rather not fix any exiting code, but as the number of comments are low, I am working on the fixes :)